webapp/src/components/Conference/Toolbar/Toolbar.tsx

```diff
@@ -21,6 +21,8 @@ export const Toolbar = (): JSX.Element => {
     effect
   } = state
 
+  const runningInElectron = navigator.userAgent.toLowerCase().includes('electron')
```

Comment:
should you use 
```ts
const desktop = await window.desktopAPI?.getAppInfo()
```
?

Reply:
It was first way I found, but I prefer yours. I will test it and make the change.

Thanks!

Reply:
What do you think about something like this?

```
  const [desktop, setDesktop] = React.useState(false)

  useEffect(() => {
    window.desktopAPI
      ?.getAppInfo()
      .then(() => {
        setDesktop(true)
      })
      .catch(console.error)
  }, [])
  ```

Reply:
@lzgrzebski I think that using `window.desktopAPI?.getAppInfo()` is a bit complex, since it returns a promise. What do you think about using the `userAgent`?

Reply:
@lzgrzebski I added the `isDesktopApp` variable to the `ConferenceContext`. Let me know what you think.